nessie/jobs/create_terms_schema.py

```diff
@@ -53,7 +53,7 @@ def create_external_schema(self):
         if not feature_flag_edl:
             # External schema is necessary when pulling term definitions from S3.
             redshift.drop_external_schema(redshift_schema)
-            sql_template = 'create_terms_schema_per_edo_db.template.sql'
+            sql_template = 'create_terms_schema.template.sql'
```

Comment:
sql_template above has no equivalent for EDL schema creation, so its name is simplified.

Reply:
Makes sense.